src/libstd/lib.rs

```diff
@@ -243,6 +244,7 @@
 #![feature(on_unimplemented)]
 #![feature(oom)]
 #![feature(optin_builtin_traits)]
+#![feature(os_extras)]
```

Comment:
Were these necessary? I thought that intra-crate usage of unstable features didn't require these annotations...